### DISCUSSION TITLE
Is there a way to have different recombination rates for males and females?

### DISCUSSION THREAD

**Discussion by mariharris** [tskit-dev/pyslim#236] [Q&A] [answered]
Hi,

I'm simulating selective sweeps on SLiM and using Pyslim to recapitate and to add neutral mutations. In SLiM it's possible to have no recombination in males and I was wondering if there was a way to do the same when I do the recapitation step on Pyslim.

Thanks!
~Mariana

**Comment by petrelharp** [comment, marked as answer]
Hah, good question. This is really a question for msprime, which does the recapitation. The short answer is "no" - but, do you really need to? When you think about tracing a lineage back from an individual's autosome, it spends half its time in males and half in females; I would think it would be very hard to impossible to tell this apart from a single, sex-averaged recombination rate, given that recaptiation starts a good bit of time ago. Are you working with a situation where you expect to see a difference?

**Reply by mariharris** [reply]
I'm comparing the autosomes with the X chromosome, and since a lineage spends 2/3 of its time on females on the X it may make a difference compared to autosomes.

**Reply by petrelharp** [reply]
Ah, yes - for the X I'd set the recombination rate for recapitation to be the average `(2/3) * (female rate) + (1/3) * (male rate)` for recapitation.

**Reply by mariharris** [reply]
okay, makes sense. Thank you!
